### ISSUE TITLE
Inserção de terrenos na versão mobile

### ISSUE THREAD

**Issue by andrebiazoti** [codigourbano/cidades-comestiveis#57]
Ontem tentei inseri um terreno e ele não apareceu no mapa.

Após inserir as informações e clicar em enviar, demorou muito para carregar e não fechou a janela.
A janela de inserção do terreno ficou aberta e não conseguia fechá-la (falta um "X" para conseguir fechá-la e voltar ao mapa).

**Comment by miguelpeixe** [comment]
Não consegui reproduzir o problema. Aconteceu mais alguma vez?
O "X" está disponível no topo da página de criação/edição de item

**Comment by andrebiazoti** [comment]
Vou ver se consigo mandar um print.

Tem como adicionar imagens aqui?

**Comment by miguelpeixe** [comment]
Sim, abaixo do campo de texto tem um botão `Attach images by dragging & dropping or selecting them. `

**Comment by andrebiazoti** [comment]
Também não estou conseguindo reproduzir. Vc mexeu em algo?
Antes a cenoura aparecia em cima do botão de "X", agora não mais.

O que acontece agora é que quando rolo pra baixo e subo de novo o X desaparece e demora um pouco a aparecer de novo.
